src/components/manage/Blocks/Image/ImageSidebar.jsx

```diff
@@ -189,6 +218,44 @@ const ImageSidebar = ({
                 });
               }}
             />
+            <TextWidget
```

Comment:
@ThomasKindermann Could you please base your PR on top of https://github.com/plone/volto/pull/2700. (Maybe change the dest. branch)
That could save you a lot of effort. We do not need to use these widgets directly.